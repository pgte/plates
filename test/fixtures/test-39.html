<ul class="breadcrumb">
  <li class="breadcrumb-element"><a class="category-name" href=""></a> <span class="divider">/</span></li>
</ul>

<div class="row-fluid content">
  
  <div class="span4" role="complementary">
    <h2>Authors</h2>
    <div class="author"></div>
  </div>
    
</div>
<!-- / row -->

<div class="doesNotExist">Hey</div>