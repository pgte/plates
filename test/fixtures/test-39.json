{
  "breadcrumb":[
    {
      "breadcrumb-element":
        [
          {
            "breadcrumb-name":"HTTP",
            "breadcrumb-url":"/categories/HTTP"
          },
          {
            "breadcrumb-name":"Client",
            "breadcrumb-url":"/categories/HTTP--Client"
          }
        ]
    }
  ],
  "author":["ABC","DEF"]
}