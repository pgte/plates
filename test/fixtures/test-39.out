<ul class="breadcrumb">
  <li class="breadcrumb-element"><a class="category-name" href="/categories/HTTP">HTTP</a> <span class="divider">/</span></li><li class="breadcrumb-element"><a class="category-name" href="/categories/HTTP--Client">Client</a> <span class="divider">/</span></li>
</ul>

<div class="row-fluid content">
  
  <div class="span4" role="complementary">
    <h2>Authors</h2>
    <div class="author">ABC</div>
    <div class="author">DEF</div>
  </div>
    
</div>

<div class="doesNotExist">Hey</div>